### DISCUSSION TITLE
Is EV3 pybricks and EV3 micropython the same?

### DISCUSSION THREAD

**Discussion by techcatgato** [pybricks/support#1684] [Q&A]
Hi, i am wondering if the EV3 pybricks 2.0 is the same as the EV3 micropython: https://education.lego.com/en-us/product-resources/mindstorms-ev3/teacher-resources/python-for-ev3/
Because their downloading have the exact same instructions.
Are they the same? Is there EV3 Pybricks? Did lego make EV3 pybricks official? i am very confused

**Comment by laurensvalk** [comment]
It's the same. We made it for LEGO Education.

If you read all the way to the end in their EULA, you'll find two names in there. :smile:

**Reply by techcatgato** [reply]
So there is no special pybricks for EV3? Does it still have improvements over the stock lego software?

**Reply by laurensvalk** [reply]
There is. Basically, "LEGO Education EV3 Python" _is_ Pybricks for EV3.

Its quite different from the stock firmware. It runs from an microSD card and runs on top of ev3dev.

**Reply by techcatgato** [reply]
> There is. Basically, "LEGO Education EV3 Python" _is_ Pybricks for EV3.
> 
> Its quite different from the stock firmware. It runs from an microSD card and runs on top of ev3dev.

Do you have any source of documentation about it? I cant find enough information on your website, and all other 3rd party toturials for for pybricks 3.0 which doesnt run on EV3

**Reply by laurensvalk** [reply]
Sure, here is the documentation: https://pybricks.com/ev3-micropython/
